zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DbReader.scala

```diff
@@ -73,14 +73,20 @@ object DbReader {
     override def getk8sNodepoolsToDeleteCandidate: Stream[F, NodepoolToScan] =
       activeNodepoolsQuery.stream.transact(xa)
 
-    override def markNodepoolAndAppStatusDeleted(id: Long): F[Unit] = {
+    override def markNodepoolAndAppStatusDeleted(nodepoolId: Long): F[Unit] = {
       val res = for {
-        _ <- updateNodepoolStatus(id, "DELETED").run
-        _ <- updateAppStatusForNodepoolId(id).run
+        _ <- updateNodepoolStatus(nodepoolId, "DELETED").run
+        _ <- updateAppStatusForNodepoolId(nodepoolId, "DELETED").run
       } yield ()
       res.transact(xa)
     }
 
-    def markNodepoolError(id: Long): F[Unit] = updateNodepoolStatus(id, "ERROR").run.transact(xa).void
+    def markNodepoolAndAppError(nodepoolId: Long): F[Unit] = {
```

Comment:
Is there a reason why we couldn't have a single method 
```scala
updateNodepoolAndAppStatus(nodepoolId: Long, status: Status): F[Unit]
```
that we can use for updating the DB statuses for both `Deleted` and `Error` cases? Nit: `update` reads better to me than `mark`.

Also, any reason why not to have a `Status` ADT instead of using `String`?

Reply:
nope..I think this was just because previous I wasn't updating them at the same time. will update